Applications/Tizen_native/CustomShortcut/inc/data.h

```diff
@@ -69,7 +77,23 @@ typedef struct appdata {
  * @param[out] length of data len
  * @retval 0 if no error
  */
-int parse_route(const char *source, char **route, char **data);
+int data_parse_route(const char *source, char **route, char **data);
+
+/**
+ * @brief get full resource path for given file.
+ * @param[in] file relative file path from resource path
+ * @param[out] full_path path of the output file
+ * @param[in] shared true if resource is in shared/res
+ */
+void data_get_resource_path(const char *file, char *full_path, bool shared);
+
+/**
+ * @brief extract data feature from given model.
+ * @param[in] ad appdata
```

Comment:
a little more descriptive? This application is going to be the reference application for the all customers of NNtrainer :).

Reply:
Yup :) will do